src/Domain/StrahlkorperTransformations.cpp

```diff
@@ -67,18 +68,36 @@ void coords_to_different_frame(
     const auto& block_id_and_coords = block_logical_coords[s].value();
     const auto& block = domain.blocks()[block_id_and_coords.id.get_index()];
 
-    if constexpr (std::is_same_v<DestFrame, ::Frame::Distorted>) {
+    if constexpr (std::is_same_v<DestFrame, ::Frame::Distorted> and
+                  std::is_same_v<SrcFrame, ::Frame::Grid>) {
       if (not block.has_distorted_frame()) {
         throw std::runtime_error(
             "Strahlkorper lies outside of distorted-frame region");
       }
       const auto& grid_to_distorted_map =
           block.moving_mesh_grid_to_distorted_map();
       x_dest = grid_to_distorted_map(x_src, time, functions_of_time);
+    } else if constexpr (std::is_same_v<DestFrame, ::Frame::Distorted> and
+                         std::is_same_v<SrcFrame, ::Frame::Inertial>) {
+      if (not block.has_distorted_frame()) {
+        throw std::runtime_error(
+            "Strahlkorper lies outside of distorted-frame region");
```

Comment:
Not a change request because you're just copying what was already done in the file, but these should probably be converted to `ERROR`s at some point.